docs/user/00-31-vpc-peering-authorization.md

```diff
@@ -92,11 +92,15 @@ For more information, see the official Google Cloud documentation on how to [gra
 ## Microsoft Azure
 <!-- VPC peering for Microsoft Azure is visible only in the Internal DRAFT version of Help Portal docs and it is not part of the Cloud Production version of Help Portal docs -->
 
-To authorize Cloud Manager in the remote subscription, Microsoft Azure requires specifying the service principal. Use the following table to identify the Cloud Manager service principal based on your Kyma landscape. Then, perform the following actions:
+To authorize Cloud Manager in the remote subscription, Microsoft Azure requires specifying the service principal. Use the following table to identify the Cloud Manager Service Principal and Application ID based on your Kyma landscape. Then, perform the following actions:
```

Comment:
```suggestion
To authorize Cloud Manager in the remote subscription, Microsoft Azure requires specifying the service principal. Use the following table to identify the Cloud Manager service principal and application ID. Then, perform the following actions:
```
I don't think we need to capitalize the service principal and application ID in a setence.